certified-connectors/Sociabble/apiDefinition.swagger.json

```diff
@@ -18,6 +18,85 @@
   "consumes": [],
   "produces": [],
   "paths": {
+    "/alerts/": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```

Comment:
@xavtesdev Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**

Reply:
@sriyen-msft @Amjed-Ayoub 
It's normal for Post "/alerts/" / "/users/{username}/customactions/"/ "/users/{username}/badges/" because they return no content ==> just status when it is ok